README.md

```diff
@@ -288,6 +289,134 @@ Example changing the spec and triggering the build via make (feel free to automa
 $ make clean && VERSION=YOURVERSIONHERE make rpm
 ```
 
+## QM sub-package KVM
+
+The QM sub-package KVM includes drop-in configuration that enables the integration of Kernel-based Virtual Machine (KVM) management into the QM (Quality Management) container environment. This configuration allows users to easily configure and manage KVM virtual machines within the QM system, streamlining virtualization tasks in containerized setups.
+
+Below example step by step:
+
+Step 1:  clone QM repo, install libvirt packages, prepare some files inside QM and start the libvirt daemon.
+
+```bash
+$host> git clone https://github.com/containers/qm.git && cd qm
+$host> make qm_dropin_mount_bind_kvm
+$host> sudo dnf install rpmbuild/RPMS/noarch/qm_mount_bind_kvm-0.6.7-1.fc40.noarch.rpm
+$host> sudo podman restart qm  # if you have qm already running
+$host> sudo dnf --installroot /usr/lib/qm/rootfs/ install virt-install libvirt-daemon libvirt-daemon-qemu libvirt-daemon-kvm -y
```

Comment:
prefer to use this one 
```
sudo dnf install qemu
```
qemu-system-x86_64 --version

Reply:
Technically, `qemu` should be containerized. This is nice for testing that the kvm device has been mounted correctly, but it is not the way to run a VM inside the QM.

Reply:
Thanks @aesteve-rh ah, yes, the whole things should be containerized,
Step 1:
 is running qemu in qm
 Step 2:
 qemu containerized in qm    
Is it? 

One more, 
Do you think we should use virt-* packages?

Reply:
Step 2 should be supported already, so we can aim for that directly.

> Do you think we should use virt-* packages?

You mean for this documented example or in general? Probably, as a response to both, it should not be necessary to use them. I agree with you in that qemu would be preferred.

Reply:
@aesteve-rh yes, should work like that. @aesteve-rh can you share more about your vision so all of us are in the same page?

Reply:
I am only saying that workloads inside the qm are supposed to be containerized. Like, the image should be stored inside the qm and launched through a quadlet file. That allows to set rules to ensure FFI, control the system's surface that the process has access to, and also allows applying SELinux labels to allow only a correct subset of operations.

For developers, it may suffice to install all those packages directly on the qm container and test it there. But imo, this is not how virtualization should be documented for users.

Reply:
@aesteve-rh completed agreed. It's just a documentation.